### PR TITLE
[TW-1114] Multi-protocol screenshots Part 2

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Updated screenshots that should include the multi-protocol dropdown menu, and other related UI changes.
* Some quick fixes--such as removing deprecated key-values, adding Contents section, updating syntax for images, and so on.

### CODE REVIEW

File: src/pages/docs/sending-requests/grpc/using-grpc-examples.md
Comment (petewitcher-postman) [inline src/pages/docs/sending-requests/grpc/using-grpc-examples.md:1]: On my localhost, the images in this file have no border when I load the page, but their borders appear after I select the images. Maybe just an anomaly on my machine?

File: src/pages/docs/sending-requests/grpc/using-service-definition.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ gRPC is a schema-driven framework. This means the client and server must adhere
...
 
-<img src="https://assets.postman.com/postman-labs-docs/grpc-docs/using-service-definition/service-definition-tab.jpeg" alt="Service definition tab">
+<img src="https://assets.postman.com/postman-docs/v10/service-definition-tab-v10.jpg" alt="Service definition tab">
 
 While creating a new request, if the service definition isn't configured, Postman prompts you to configure it when you select the **Method selection** dropdown. This interface serves as a quick way to add your service definition and move on to invoking your request. From this menu, you can select a protobuf API available in your workspace or import a new one by dragging a `.proto` file from your local system and dropping it into Postman.
 
-<img src="https://assets.postman.com/postman-labs-docs/grpc-docs/using-service-definition/method-selection-dropdown.jpeg" alt="Configure service definition from method selection dropdown">
+<img src="https://assets.postman.com/postman-docs/v10/method-selection-dropdown-v10.jpg" alt="Configure service definition from method selection dropdown">
```
Comment (petewitcher-postman) [inline src/pages/docs/sending-requests/grpc/using-service-definition.md:21]: This image has no border until I select it.

File: src/pages/docs/sending-requests/grpc/using-service-definition.md [hunk 2]
```diff
@@ -36,17 +36,17 @@ To load a service definition to the request, you can either use server reflectio
 
 If enabled by the server, server reflection can be the easiest way to add the service definition to the client. It fetches the latest information available in the schema from the server without you having to manually load a `.proto` file or create a protobuf schema. Enter the URL and Postman loads the service definition automatically.
 
-<img src="https://assets.postman.com/postman-labs-docs/grpc-docs/using-service-definition/using-server-reflection.gif" alt="Using service definition">
+<img src="https://assets.postman.com/postman-docs/v10/using-server-reflection-v10.jpg" alt="Using service definition">
```
Comment (petewitcher-postman) [inline src/pages/docs/sending-requests/grpc/using-service-definition.md:39]: This image has no border until I select it.

File: src/pages/docs/sending-requests/grpc/using-service-definition.md [hunk 3]
```diff
@@ -70,11 +70,11 @@ This will create a new protobuf API from the `.proto` file and use it in the req
 
 > You can also import the `.proto` file as an update to an existing API. While on the **Import as API** interface, you can select an API from the list of available APIs in the workspace and add the `.proto` file as a new version. You can also replace it with an existing version by selecting an existing version from the list.
 
-<img src="https://assets.postman.com/postman-labs-docs/grpc-docs/using-service-definition/import-proto-file.gif" alt="Importing a proto file">
+<img src="https://assets.postman.com/postman-docs/v10/import-proto-file-v10.jpg" alt="Importing a proto file">
```
Comment (petewitcher-postman) [inline src/pages/docs/sending-requests/grpc/using-service-definition.md:73]: This image has no border until I select it.

**Review by petewitcher-postman — APPROVED** [review]
LGTM! A few images had no borders until I selected them, but maybe that's isolated to my computer. I did clear the browser cache (shift + refresh) with the same results.

File: src/pages/docs/sending-requests/grpc/using-grpc-examples.md
Comment (MatthewPinsker-postman) [reply src/pages/docs/sending-requests/grpc/using-grpc-examples.md:1]: Thanks for bringing this up! I just tested all screenshots you mentioned, and I'm seeing the borders before I select the images.

@mark-anthony-postman Are you experiencing the same issue?

Comment (mark-anthony-postman) [reply src/pages/docs/sending-requests/grpc/using-grpc-examples.md:1]: I've noticed a little weirdness with borders. At first I wasn't seeing some in a few cases, but after I resized my browser window I did. I think for whatever reason some of the borders are just very thin, so maybe that's why they're not showing depending on how the image is scaled.

File: src/pages/docs/sending-requests/grpc/using-service-definition.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/sending-requests/grpc/using-service-definition.md:21]: I do see a very thin border

File: src/pages/docs/sending-requests/grpc/using-service-definition.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/sending-requests/grpc/using-service-definition.md:39]: I do see a very thin border

File: src/pages/docs/sending-requests/grpc/using-service-definition.md (hunk 3 above)
Comment (mark-anthony-postman) [reply src/pages/docs/sending-requests/grpc/using-service-definition.md:73]: I do see a very thin border

**Review by mark-anthony-postman — APPROVED** [review]
LGTM! Thanks for redoing so many screenshots so quickly! 🌟

**Pull request merged**